ops.go

```diff
@@ -367,15 +373,35 @@ func cmdListPackages(cmd *cobra.Command, args []string) {
 
 	table.SetRowLine(true)
 
-	for key, value := range pacakges {
+	var r *regexp.Regexp
+	var filter bool
+	if len(searchRegex) > 0 {
+		filter = true
+		r, err = regexp.Compile(searchRegex)
+		if err != nil {
+			panic(err)
```

Comment:
Don't need to panic here, if regex is not valid continue as if there is no filter.